cmd/cluster-capi-operator/main.go

```diff
@@ -298,24 +310,9 @@ func setupInfraClusterReconciler(mgr manager.Manager, platform configv1.Platform
 	}
 }
 
-func setupWebhooks(mgr ctrl.Manager, platform configv1.PlatformType) {
-	if err := (&webhook.CoreProviderWebhook{}).SetupWebhookWithManager(mgr); err != nil {
-		klog.Error(err, "unable to create webhook", "webhook", "CoreProvider")
-		os.Exit(1)
-	}
-
-	if err := (&webhook.InfrastructureProviderWebhook{Platform: platform}).SetupWebhookWithManager(mgr); err != nil {
-		klog.Error(err, "unable to create webhook", "webhook", "InfrastructureProvider")
-		os.Exit(1)
-	}
-
```

Comment:
These were upstream cluster-api-operator CRs validations. I cleaned those up now that we don't use it anymore.